client/src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App(): JSX.Element {
                   <Switch>
                     <Route exact path="/login" component={Login} />
                     <Route exact path="/signup" component={Signup} />
+                    <Route exact path="/home" component={HomePage} />
```

Comment:
Optional: I think we can update the route to `/`, that way this will be the first page that is rendered when the app is run :)